formidling/src/main/java/no/nav/ung/sak/formidling/vedtak/regler/strategy/EndringInntektUtenReduksjonStrategy.java

```diff
@@ -0,0 +1,55 @@
+package no.nav.ung.sak.formidling.vedtak.regler.strategy;
+
+import jakarta.enterprise.context.Dependent;
+import jakarta.inject.Inject;
+import no.nav.fpsak.tidsserie.LocalDateTimeline;
+import no.nav.ung.kodeverk.behandling.aksjonspunkt.AksjonspunktDefinisjon;
+import no.nav.ung.kodeverk.dokument.DokumentMalType;
+import no.nav.ung.sak.behandlingslager.behandling.Behandling;
+import no.nav.ung.sak.behandlingslager.behandling.aksjonspunkt.Aksjonspunkt;
+import no.nav.ung.sak.formidling.innhold.EndringRapportertInntektUtenReduksjonInnholdBygger;
+import no.nav.ung.sak.formidling.vedtak.regler.IngenBrevÅrsakType;
+import no.nav.ung.sak.formidling.vedtak.regler.VedtaksbrevEgenskaper;
+import no.nav.ung.sak.formidling.vedtak.resultat.DetaljertResultat;
+import no.nav.ung.sak.formidling.vedtak.resultat.DetaljertResultatType;
+
+@Dependent
+public final class EndringInntektUtenReduksjonStrategy implements VedtaksbrevInnholdbyggerStrategy {
+
+
+    private final EndringRapportertInntektUtenReduksjonInnholdBygger endringRapportertInntektUtenReduksjonInnholdBygger;
+
+    @Inject
+    public EndringInntektUtenReduksjonStrategy(EndringRapportertInntektUtenReduksjonInnholdBygger endringRapportertInntektUtenReduksjonInnholdBygger) {
+        this.endringRapportertInntektUtenReduksjonInnholdBygger = endringRapportertInntektUtenReduksjonInnholdBygger;
+    }
+
+    @Override
+    public VedtaksbrevStrategyResultat evaluer(Behandling behandling, LocalDateTimeline<DetaljertResultat> detaljertResultat) {
+        boolean harUtførtKontrollerInntekt = behandling.getAksjonspunkter().stream()
```

Comment:
Er vi sikre på at det er dette som skal vere styrende? Slik eg forstod det på Ina så skulle dette brevet sendes ut for dei som 
- ikkje allerede har fått oppgave
- har rapportert inntekt > 0
- har fastsatt reduksjon lik 0 (altså enten fastsatt av saksbehandler eller automatisk)

Eg tenkjer dersom man i utgangspunktet er interessert i eit spesifikt funksjonelt case, så burde man ikkje knytte det til aksjonspunkt sidan utledelsen av aksjonspunkt kan endre seg (og då er det fort gjort å ikkje tenkje på denne avhengigheten).